### DISCUSSION TITLE
Collation of Useful Resources 😎✨

### DISCUSSION THREAD

**Discussion by Chieto08** [AISaturdaysLagos/cohort_structure#8] [General]
Hello everyone, this is a forum where we can turn to, in case we have blockers or questions or just seeking out useful resources 😎. Please if you have questions or answers or wants to contribute useful resources😎, feel free to drop by!🙂

**Comment by Chieto08** [comment]
Here's a very good tutorial on [Github guide for beginners](https://youtube.com/watch?v=iv8rSLsi1xo&amp;feature=share) in case you need a guide to help you kick-off.😎

**Reply by marthaakinlolu** [reply]
Thanks

**Comment by shadynen** [comment]
Thanks

**Comment by ModibboMuawiyya** [comment]
Here are a few cheatsheets for regex in python. 

https://www.geeksforgeeks.org/python-regex-cheat-sheet/
https://www.dataquest.io/blog/regex-cheatsheet/

**Comment by tejuafonja** [comment]
Markdown cheatsheet: 
https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet

**Comment by LaseTawak** [comment]
Here's a link to a cheat sheet for Regex
http://www.rexegg.com/regex-quickstart.html#ref
